crates/swc/tests/errors/lints/quotes/valid-cases-backtick/input.js

```diff
@@ -0,0 +1,52 @@
+var s = "double quotes string";
+
+// valid cases
+
+var foo = `bar`;
+var foo = `bar 'baz'`;
+var foo = `bar \"baz\"`;
+var foo = 1;
+var foo = "a string containing `backtick` quotes";
+var foo = `bar 'foo' baz` + `bar`;
+
+() => {
+    "use strict";
+    var foo = `backtick`;
+};
+
+() => {
+    "use strict";
+    "use strong";
+    "use asm";
+    var foo = `backtick`;
+};
+
+// TODO: "error: 'import', and 'export' cannot be used outside of module code"
```

Comment:
@kdy1 why tests throws me this error? I was dive in code of test and it marks code as module

Reply:
I don't know

Reply:
Maybe `IsModule` is used?

Reply:
Yes, It used